app/javascript/components/Product/ProductCard.tsx

```diff
@@ -0,0 +1,84 @@
+import { classNames } from "$app/utils/classNames";
+import * as React from "react";
+
+export const ProductCardArticle = ({
```

Comment:
```suggestion
export const ProductCard = ({
```

I think the suffix is more confusing than helpful